_posts/technotes/2017-01-03-jsonlite-12.md

```diff
@@ -0,0 +1,50 @@
+---
+name: jsonlite-12
+layout: post
+title: "Update jsonlite 1.2"
+authors:
+  - name: Jeroen Ooms
+    url: https://ropensci.org/about/
```

Comment:
@jeroenooms you don't need to include a URL - we parse the name field and include link automatically https://github.com/ropensci/roweb/blob/master/_layouts/post.html#L14 for ropensci people